toronto/src/pages/UserListPage.jsx

```diff
@@ -0,0 +1,96 @@
+import UserList from '@/components/organisms/UserList';
+import { useCallback, useEffect, useState } from 'react';
+import styled from 'styled-components';
+import InputBar from '@/components/molecules/InputBar';
+import axios from 'axios';
+import Skeleton from '@/components/atoms/Skeleton';
+
+const renderSkeleton = () => {
+  const skeletons = [];
+
+  for (let i = 0; i < 10; i++) {
+    skeletons.push(<Skeleton.Box key={i} width={'100%'} height={'100%'} />);
+  }
+
+  return skeletons;
+};
+
+const UserListPage = () => {
+  const [users, setUsers] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const initialUsers = useCallback(async () => {
+    setIsLoading(true);
+
+    const res = await axios.get(
+      `${process.env.REACT_APP_END_POINT}/users/get-users`,
+    );
+
+    setUsers([...res.data]);
+    setIsLoading(false);
+  }, []);
+
+  useEffect(() => {
+    initialUsers();
+  }, [initialUsers]);
+
+  const handleSubmit = useCallback(
+    async (value) => {
+      if (!value) {
+        initialUsers();
+        return;
+      }
+
+      setIsLoading(true);
+
+      const res = await axios.get(
+        `${process.env.REACT_APP_END_POINT}/search/users/${value}`,
+      );
+
+      setUsers([...res.data]);
+      setIsLoading(false);
+    },
+    [initialUsers],
+  );
+
+  return (
+    <Container>
+      <Wrapper>
+        <InputBar
+          placeholder={'유저 검색'}
+          buttonType={'inside'}
+          onSubmit={(value) => handleSubmit(value)}
+        />
+      </Wrapper>
+      <GridContainer>
+        <UserList users={users} />
+        {isLoading ? renderSkeleton() : undefined}
+      </GridContainer>
+    </Container>
+  );
+};
+
+export default UserListPage;
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+  height: 100%;
+  margin: 0 auto;
+`;
+
+const Wrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+`;
+
+const GridContainer = styled.ul`
+  display: grid;
+  padding: 0;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  grid-template-rows: repeat(auto-fit, minmax(300px, 1fr));
+  grid-auto-rows: minmax(300, 1fr);
+  grid-auto-columns: minmax(300, 1fr);
```

Comment:
minmax 내부에 `300`을 `300px` 로 고쳐야 할 것 같습니다! 현재는 값을 인식하지 못해서 모든 `Skeleton` 컴포넌트가 제대로 출력되지 않습니다 😢 auto-rows와 auto-columns 수정 부탁드립니다 🙇 
```suggestion
  grid-auto-columns: minmax(300px, 1fr);
```

Reply:
헉 그렇네요.... 첫 줄에 뜨는것만 확인하고 신나서 올렸는데 2번째 라인부터 출력이 안 되는 버그가 있었네용 😥

해당 내용도 수정 완료했습니다!